setup.py

```diff
@@ -46,6 +46,7 @@ def package_files(package_dir, subdirectory):
                 'scripts/flag_all.py', 'scripts/throw_away_flagged_antennas.py', 'scripts/select_spw_ranges.py',
                 'scripts/multiply_gains.py', 'scripts/lstbin_simple.py', 'scripts/subselect.py'],
     'package_data': {'hera_cal': data_files},
+    'python_requires': '>3.8',
```

Comment:
shouldn't it be >=3.9? Otherwise, I think python 3.8.10 or whatever would qualify

Reply:
Yes, I think >=3.9 is the way to go